app/src/main/java/com/example/mobiledger/domain/usecases/CategoryUseCase.kt

```diff
@@ -53,4 +61,46 @@ class CategoryUseCaseImpl(private val categoryRepository: CategoryRepository) :
     override suspend fun getAllMonthlyCategories(monthYear: String): AppResult<List<MonthlyCategorySummary>> {
         return categoryRepository.getAllMonthlyCategories(monthYear)
     }
+
+    override suspend fun getMonthlyCategoryTransactionReferences(
```

Comment:
Why is this in Usecase?

Reply:
Faltu fokat k usecase bnaa rkhe h..unhe b kch kaam de dia krne ko xD